main/glossary/README.md

```diff
@@ -67,151 +102,362 @@ synchronously compared for structural equivalence.
+- A handle used to refer to this invitation.
+- A description of this invitation's purpose.
+
+## InvitationIssuer
+
+Since [invitations](#invitation) are special cases of [payments](#payment), invitations 
+must have a dedicated [issuer](#issuer), which is the InvitationIssuer.
+
+Zoe has a single `InvitationIssuer` for its entire lifetime. By having a reference to Zoe, 
+a user can get the `InvitationIssuer`. This lets them claim any invitation they 
+receive from someone else by calling `E(invitationIssuer).claim()` with the untrusted
+invitation as the argument. During the claiming process, the invitationIssuer validates
+the invitation. A successful claim also means that invitation is exclusively yours.
 
 ## Issuer
-Issuers are linked to a single [mint](#mint) and vice versa, so each issuer works
+Issuers are a one-to-one relationship with both a [mint](#mint) and a [brand](#brand), so each issuer works
 with one and only one asset type, such as only working with quatloos or only working
-with moola. They can create empty [purses](#purse) and [payments](#payment) for
-their asset type, but cannot mint new [amounts](#amounts). 
+with moola. This association cannot change to another type. 
 
-Issuers can also transform
+Issuers can create empty [purses](#purse) and [payments](#payment) for
+their asset type, but cannot mint new [amounts](#amount). Issuers can also transform
 payments of their asset type (splitting, combining, burning, and exclusively claiming
 payments). An issuer from a trusted source can determine if an untrusted payment of
 its asset type is valid. 
 
-See the [ERTP Guide's Issuer section](/ertp/guide/issuers-and-mints.md)
-and the [ERTP API's Issuer section](/ertp/api/issuer.md) for more information.
+For more information, see the [ERTP Guide's Issuer section](/ertp/guide/issuers-and-mints.md)
+and the [ERTP API's Issuer section](/ertp/api/issuer.md).
+
+## Keywords
+
+Keywords are unique identifiers per contract. They tie together the [proposal](#proposal), [payments](#payment) 
+to be [escrowed](#escrow), and [payouts](#payout) to the user by serving as keys for key-value pairs in various 
+records with values of [amounts](#amount), [issuers](#issuer), etc.
 
 ## Mint
-In ERTP, mints create digital assets and are the only objects with the authority to do so. 
-Access to a mint gives you the power to create more digital assets of its type at will. Mints
-can only create one type of asset. 
+Agoric has two mint objects, *ERTP mints* and *Zoe Contract Facet mints (ZCFMints)*. They both create
+digital assets. Which mint type creates an asset doesn't matter; quatloos created by an ERTP mint are 
+indistinguishable from quatloos created by a ZCFMint. 
 
-Mints are [issuer's](#issuer) admin facets, and there is a one-to-one relationship between an issuer and
-its mint.
+- ERTP mints create digital assets asynchronously and are the only ERTP objects with the authority to do so. 
```

Comment:
No, minting is a synchronous method. This is not the distinction between ZCFMints and the underlying ERTP mint. The distinction is that the ZCFMint can mint assets that get associated with a seat without having to escrow payments, and burn assets that used to be associated with a seat without having to payout assets.